config/jail.conf

```diff
@@ -839,3 +839,13 @@ logencoding = utf-8
 # See "haproxy-http-auth" filter for a brief cautionary note when setting
 # maxretry and findtime.
 logpath  = /var/log/haproxy.log
+
+
+[zoneminder]
+# Zoneminder HTTP/HTTPS web interface auth
+# Logs auth failures to apache2 error log
+enabled = false
```

Comment:
once again -- no need for 'enabled=false' -- just see the other jail definitions